hyperv-samples/hcs-samples/JSON_files/HCS_Schema[Windows_10_SDK_version_1809].json

```diff
@@ -2594,6 +2708,13 @@
                 "Unknown"
             ]
         },
+        "OperationType": {
+            "type": "string",
+            "description": "TODO: Replace usage of ActiveOperation with this?",
```

Comment:
Remove the TODO. Is there a way we can update the mars files to avoid this other than removing the comment?